### ISSUE TITLE
Missing Instructions on installing Supervisor for HA Core + Incorrect python command

### ISSUE THREAD

**Issue by raviwarrier** [home-assistant/home-assistant.github.io#17523]
### Feedback

Hi there, I installed HA core using the instructions on the page only to find that after all that effort there was no Supervisor and I couldn't find any instructions on how to install it separately.
Second, you have a command using python3.8 -m venv, but python3.8 is not the default python version.

Could you please fix both these instruction sets? Thx.

W.

### URL

https://www.home-assistant.io/installation/raspberrypi/

### Version

2021.4.5

**Comment by hmmbob** [comment]
Home Assistant Supervised is only supported on generic Linux devices running Debian, and not on Raspberry Pi's as you can see in the docs: https://www.home-assistant.io/installation/

So, if you install _HA Core_, you will get _HA Core_ and not the _Supervised_ (or _HA OS_) version 🤷🏻 
See the same page for the differences between the installation methods and the supported installation methods per platform.

As for the `python3.8`  command, see the very first big blue NOTE box in the ["Install Home Assistant Core"  section](https://www.home-assistant.io/installation/raspberrypi#install-home-assistant-core) you referenced:

> Prerequisites
This guide assumes that you already have an operating system setup and have installed Python 3.8 (including the package python3-dev) or newer.


Basically, I'd recommend you to follow the "Install HA OS" instructions: https://www.home-assistant.io/installation/raspberrypi#install-home-assistant-operating-system

**Comment by frenck** [comment]
Yep, adding to that, we consider installation Home Assistant Core in a pure Python environment an advanced use case for more experienced users in e.g., Python.

Depending on your platform/OS/Linux flavor, the commands may differ. We assume one is able to adopt to that in those cases.

If this isn't the case for you, we don't recommend on using that installation method, but maybe use the OS or Container installation methods instead.


If you are in need of additional help/support, you should try our [Community Forum](https://community.home-assistant.io) or join our [Discord chat server](https://discord.gg/c5DvZ4e).

Thanks! 👍

../Frenck 🚂
